frontend/src/components/common/form/RadioGroup.tsx

```diff
@@ -0,0 +1,135 @@
+import classNames from 'classnames';
+import { getIn, useFormikContext } from 'formik';
+import React, { ReactNode } from 'react';
+import Form from 'react-bootstrap/Form';
+import { FormCheckProps } from 'react-bootstrap/FormCheck';
+import styled from 'styled-components';
+
+import { InlineFlexDiv } from '../styles';
+import TooltipIcon from '../TooltipIcon';
+import { DisplayError } from './DisplayError';
+
+type RequiredAttributes = {
+  field: string;
+  radioValues: { radioLabel: ReactNode; radioValue: string }[];
+};
+
+type OptionalAttributes = {
+  /** The form component label to display before the checkbox */
+  label?: string;
+  /** The form component label to display after the checkbox */
+  postLabel?: string;
+  /** The underlying HTML element to use when rendering the FormControl */
+  as?: React.ElementType;
+  /** Short hint that describes the expected value of an <input> element */
+  placeholder?: string;
+  /** Adds a custom class to the input element of the <Input> component */
+  className?: string;
+  /** Whether the field is required. Makes the field border blue. */
+  required?: boolean;
+  /** Specifies that the HTML element should be disabled */
+  disabled?: boolean;
+  /** Use React-Bootstrap's custom form elements to replace the browser defaults */
+  custom?: boolean;
+  /** style to use for the formgroup wrapping the inner element */
+  innerClassName?: string;
+  /** override the display of the component, default is checkbox. Select radio to display this checkbox as two radio buttons. */
+  type?: string;
+  /** Optional tool tip message to add to checkbox */
+  toolTip?: string;
+  /** id for tooltip */
+  toolTipId?: string;
+};
+
+// only "field" is required for <RadioGroup>, the rest are optional
+export type RadioGroupProps = FormCheckProps & OptionalAttributes & RequiredAttributes;
+
+/**
+ * Formik-connected <RadioGroup> form control
+ */
+export const RadioGroup = ({
```

Comment:
new radio group component. takes in an array of label/value pairs. the selected radio button will inject the corresponding value into a single field name.